app/controllers/kudos_controller.rb

```diff
@@ -60,7 +60,7 @@ def create
 
     kudo = Kudo.new(
       giver_id: giver_id,
-      receiver_id: receiver.id,
+      receiver_ids: [receiver.id],
```

Comment:
Confused, should this be `receivers`?

Reply:
I believe `receivers` refers to the actual `User` models, whereas this is just using the Users' ids. I think passing in an array for `receiver_ids` auto magically creates the multiple `KudoReceipts`s to join between `Kudo` and `User` (@joecox please confirm)

Reply:
Gotcha, thanks for the explanations @joecox and @nnsookwon!
Rails magic continues to befuddle me. =_=

The logical separation sounds good, thanks for the smaller PRs for us! :D